ivy/data_classes/array/array.py

```diff
@@ -792,9 +792,19 @@ def __bool__(self):
         return self._data.__bool__()
 
     def __dlpack__(self, stream=None):
-        return self._data.__dlpack__()
+        # Not completely supported yet as paddle and tf
```

Comment:
I think we can just add the comment once in `__dlpack__` as `__dlpack_device__` is right below it anyway. Rest looks good to merge, thanks @abdulasiraj 😄